core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.constants.ts

```diff
@@ -21,9 +21,10 @@
 
 export class TopicLandingPageConstants {
   public static TOPIC_LANDING_PAGE_DATA = {
-    maths: {
+    math: {
       fractions: {
         topic_title: 'Fractions',
+        topic_tagline: 'Add, Subtract, Multiply and Divide',
```

Comment:
Define this for other pages as well with an empty string. (And remove the type checking in the controller)

Reply:
Done. (We still need to check if the tagline is empty, though, but I guess that should be alright)